<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>291</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Pending_Layout">
       <item>
        <widget class="QFrame" name="Pending">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="PendingButton">
               <property name="text">
                <string>Pending Proposals</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="DatePublished_Pending">
            <property name="text">
             <string>Date Published</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QTableWidget" name="PendingTable">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Events</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>321</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_Upcoming_Resched">
       <item>
        <widget class="QStackedWidget" name="UpcomingReschedule">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_5">
          <widget class="QFrame" name="UpcomingEvents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>321</width>
             <height>461</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="UpcomingButton_2">
                 <property name="text">
                  <string>Upcoming Events</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="RejectedButton">
                 <property name="text">
                  <string>Rejected Proposals</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="DatePublished_Events_6">
              <property name="text">
               <string>Date Published</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="Events_table_6">
              <row>
               <property name="text">
                <string>7:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>8:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>9:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>11:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>12:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>1:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>5:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>6:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>7:00 PM</string>
               </property>
              </row>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_6">
          <widget class="QFrame" name="RejectedProposals">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>321</width>
             <height>461</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QPushButton" name="UpcomingButton">
                 <property name="text">
                  <string>Upcoming Events</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="RejectedButton_2">
                 <property name="text">
                  <string>Rejected Proposals</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="DatePublished_Events_7">
              <property name="text">
               <string>Date Published</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="Events_table_7">
              <row>
               <property name="text">
                <string>7:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>8:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>9:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>11:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>12:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>1:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>5:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>6:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>7:00 PM</string>
               </property>
              </row>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ViewAttendanceButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>530</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>View Attendance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RequestRescheduleButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>530</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Reschedule</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RequestEventProposalButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Event Proposal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
